c8run/package.go

```diff
@@ -0,0 +1,153 @@
+package main
+
+import (
+	"c8run/internal/archive"
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"runtime"
+)
+
+func Clean(camundaVersion string, elasticsearchVersion string) {
+	os.RemoveAll("elasticsearch-" + elasticsearchVersion)
+	os.RemoveAll("camunda-zeebe-" + camundaVersion)
+
+	_, err := os.Stat(filepath.Join("log", "camunda.log"))
+	if errors.Is(err, os.ErrNotExist) {
+		os.Remove(filepath.Join("log", "camunda.log"))
+	}
+	_, err = os.Stat(filepath.Join("log", "connectors.log"))
+	if errors.Is(err, os.ErrNotExist) {
+		os.Remove(filepath.Join("log", "connectors.log"))
+	}
+	_, err = os.Stat(filepath.Join("log", "elasticsearch.log"))
+	if errors.Is(err, os.ErrNotExist) {
+		os.Remove(filepath.Join("log", "elasticsearch.log"))
+	}
+}
+
+func downloadAndExtract(filePath, url, extractDir string, extractFunc func(string, string) error) error {
+	_, err := os.Stat(filePath)
+	if errors.Is(err, os.ErrNotExist) {
+		err = archive.DownloadFile(filePath, url)
+		if err != nil {
+			return err
+		}
+		fmt.Println("File downloaded successfully to " + filePath)
+	}
+
+	_, err = os.Stat(extractDir)
+	if errors.Is(err, os.ErrNotExist) {
+		err = extractFunc(filePath, ".")
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func PackageWindows(camundaVersion string, elasticsearchVersion string) {
+	elasticsearchUrl := "https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-" + elasticsearchVersion + "-windows-x86_64.zip"
+	elasticsearchFilePath := "elasticsearch-" + elasticsearchVersion + ".zip"
+	camundaFilePath := "camunda-zeebe-" + camundaVersion + ".zip"
+	camundaUrl := "https://github.com/camunda/camunda/releases/download/" + camundaVersion + "/" + camundaFilePath
+	connectorsFilePath := "connector-runtime-bundle-" + camundaVersion + "-with-dependencies.jar"
+	connectorsUrl := "https://repo1.maven.org/maven2/io/camunda/connector/connector-runtime-bundle/" + camundaVersion + "/" + connectorsFilePath
+
+	Clean(camundaVersion, elasticsearchVersion)
+
+	err := downloadAndExtract(elasticsearchFilePath, elasticsearchUrl, "elasticsearch-"+elasticsearchVersion, archive.UnzipSource)
+	if err != nil {
+		panic(err)
```

Comment:
package functions should return an error and let main panic... otherwise this is difficult to test

Reply:
https://github.com/camunda/camunda/pull/24040/commits/4aa3968f155af3b001d67e383aedef54f6eb22af

the above  commit addresses this comment